php-fpm/Dockerfile

```diff
@@ -42,6 +42,13 @@ RUN curl -sS https://getcomposer.org/installer | tee composer-setup.php \
     && chmod +x composer.phar && mv composer.phar /usr/bin/composer \
     && ln -s /usr/bin/php8 /usr/local/bin/php
 
+ARG SYMFONY_CLI_VERSION
+RUN wget https://github.com/symfony/cli/releases/download/v$SYMFONY_CLI_VERSION/symfony_linux_amd64.gz \
+    && gzip -d symfony_linux_amd64.gz \
+    && mv symfony_linux_amd64 symfony \
```

Comment:
You could move it to /usr/local/bin/ directly so you will be able to remove the last move line ;)